benchmarks/.clang-format

```diff
@@ -0,0 +1,65 @@
+---
```

Comment:
Is it possible to put these files in the root directory? We can also nested them under `src/` if there is some complication